src/store/epics/index.js

```diff
@@ -0,0 +1,64 @@
+import {
+    Observable,
+    of
+} from 'rxjs';
+import {
+    map,
+    pluck,
+} from 'rxjs/operators';
+import {
+    MESSAGE_POLLING,
+    MESSAGE_SENT,
+    MESSAGE_POLLING_START,
+    MESSAGE_POLLING_STOP
+} from "../actions/types";
+
+import fakePaymentData from './tmpFakeData';
+
+import { ajax } from 'rxjs/ajax';
+
+const ENDPOINT = '/api/payments_light';
+
+const fakeResp = fakePaymentData;
+
+const getTransactionInfo = (payments) => {
+    // restful url is not ready yet.
+
+    // return ajax({
+    //     url: ENDPOINT,
+    //     method: 'POST',
+    //     headers: {
+    //         'Content-Type': 'application/json',
+    //         'rxjs-custom-header': 'Rxjs'
+    //     },
+    //     body: {
+    //         payments
+    //     }
+    // })
+
+    return of([...fakeResp])
+};
+
+const pluckState = (state$, state) => state$.pipe(pluck(state));
+
+const paymentsMonitoredEpic = (action$, state$) => {
```

Comment:
Need some comments here. What does this function do?

Pipe getTransactionInfo(payments) response to where?